axonserver/src/main/java/io/axoniq/axonserver/localstorage/DataFeatcherSchedulerProvider.java

```diff
@@ -0,0 +1,29 @@
+package io.axoniq.axonserver.localstorage;
+
+import org.springframework.scheduling.concurrent.CustomizableThreadFactory;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.function.Supplier;
+
+/**
+ * @author Stefan Dragisic
+ * @author Sara Pellegrini
+ * @since 4.5.3
+ */
+
+public class DataFeatcherSchedulerProvider implements Supplier<ExecutorService> {
```

Comment:
Typo in class name. 

This class is fine for an urgent bug-fix, but we should really get rid of 'global variables' as soon as we can.